DataConnectors/GCP/README.md

```diff
@@ -0,0 +1,56 @@
+# GCP Microsoft Sentinel Connector
+
+## Introduction
+
+GCP Sentinel connector is a platform for ingesting GCP service logs into Azure Sentinel. Currently supported logs include: Audit logs.
+
+This connector requires that each GCP service will publish its logs to a pub/sub. In addition you must configure workload identity pool and workload identity provider with permissions for the connector to receive an access token as well as a service account with permissions for the connector to read the logs from the pub/sub subscription.
+
+More information on the connector and configuration instructions can be found on the Azure Sentinel data connector page in the Azure portal.
+
+## Configuration process
+
+This set of Terraform scripts can be used to automatically configure the necessary resources.
+
+At a high level, these scripts do the following:
+
+### `GCPInitialAuthenticationSetup.tf` script:
+
+1. Create a workload identity pool (only if marked as not exist) with azure tenant id as its id.
+2. Create a workload identity provider with Sentinel's application settings.
+3. Create a service account with permissions to read from pub/sub subscription.
+
+### `Script that configures data to subscription` (for example GCPAuditLogsSetup.tf script):
+
+1. Create a topic.
+2. Create a subscription.
+3. Configure the GCP service (for example: auditlogs) to export logs to the pub/sub.
+   
+## Using the scripts
```

Comment:
Suggestion:
* Use
* Bullets

Or
1. Use
2. Numbered
3. list

for this section

Reply:
Added numbered list